api/hyp3_api/auth.py

```diff
@@ -0,0 +1,29 @@
+import time
+from os import environ
+
+import jwt
+
+
+def decode_token(token, required_scopes):
+    try:
+        jwt.decode(token, environ['AUTH_PUBLIC_KEY'], algorithms=environ['AUTH_ALGORITHM'])
+    except (jwt.DecodeError, jwt.ExpiredSignatureError):
+        return {
+            'active': False
+        }
+    return {
+        'active': True
+    }
+
+
+def get_mock_jwt_cookie(user, expires_in=100):
```

Comment:
I like units on variable names where possible, `lifetime_in_seconds` or the like

Reply:
Sounds good, I just stole that from Datapool